### PR TITLE
Refactoring of news feed cards according to the designs

### PR DESCRIPTION
**Pull request by BirajPaul**
Main change includes `... more` text component.

Update: Fixed Mariia's review comments. Gif updated below
Short gif:

card-refactor

### CODE REVIEW

File: front-end/src/components/newsfeed-view/news-list/card-group/news-card/FacebookCard.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+        <div className={styles['content']}>
+            <div className={styles['content-source']}>
+                <img src={data.thumbnail} alt='Facebook thumbnail' />
+                <div>{data.author}</div>
+            </div>
+            <div className={styles['description']}>
+                <ExpandableContent 
+                    maxLine={4} 
+                    description={data.description} 
+                    ellipsis='... more'
```
Comment (mariiu) [inline front-end/src/components/newsfeed-view/news-list/card-group/news-card/FacebookCard.js]: do we have a finnish alternative to 'more' and is it displayed fine? Or will we have 'more' for all language settings? Should ask Laura probably

Comment (BirajPaul) [reply front-end/src/components/newsfeed-view/news-list/card-group/news-card/FacebookCard.js]: Good question. Will get it clarified by Laura or Daniella.

File: front-end/src/components/newsfeed-view/news-list/card-group/news-card/TwitterCard.js [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+        <div className={styles["service-source"]}>
+            {data.source}
+        </div>
+        <div className={styles['content']}>
+            <div className={styles['content-source']}>
+                <img src={data.thumbnail} alt='Twitter thumbnail' />
+                <div>
+                    {data.author}
+                    <br />
+                    <span className={styles['twitter-handle']}>@{data.userhandle}</span>
```
Comment (mariiu) [inline front-end/src/components/newsfeed-view/news-list/card-group/news-card/TwitterCard.js]: need to inform Shamnad that we will need this userhandle field as well, right now we have only author name and thumbnail

Comment (BirajPaul) [reply front-end/src/components/newsfeed-view/news-list/card-group/news-card/TwitterCard.js]: Yes, Indeed. I will inform him.

File: front-end/src/components/newsfeed-view/news-list/card-group/news-card/elements/expandable-content.module.scss [hunk 3]
```diff
@@ -0,0 +1,17 @@
+$more-text-width: 12 * 1.04rem;
+
+.expandable-div {
+    overflow: hidden;
+    position: relative;
+
+    .more-text  {
+        // background: #fff;
+        background: linear-gradient(to right, rgba(255, 255, 255, 0), rgba(255, 255, 255, 1) 40%);
```
Comment (StormBlade97) [inline front-end/src/components/newsfeed-view/news-list/card-group/news-card/elements/expandable-content.module.scss]: accept background color from react props

**Pull request merged**